### DISCUSSION TITLE
Adding private transactions to Rainbow Wallet

### DISCUSSION THREAD

**Discussion by RonanKMcGovern** [rainbow-me/rainbowkit#820] [Q&A]
I'm considering building on Rainbow Wallet to add private transaction capabilities.

I'm interested in talking to devs familiar with the Rainbow wallet codebase and have funds to support development.

Thanks, Ronan

**Comment by nickbytes** [comment]
Hi @RonanKMcGovern – what do you mean by private transactions?

**Reply by RonanKMcGovern** [reply]
Transactions where the amounts and addresses are shielded (i.e. not public). The approach I'm taking is to use Aztec (zk-rollup approach). The app will be focused on small transfer sizes (roughly replicating cash).

**Reply by nickbytes** [reply]
If the chain is EVM-compatible, you should be good to go. Right now, we're not supporting non-EVM-compatible chains.
